backend/de.metas.business/src/main/java/de/metas/payment/interceptor/PaymentModuleInterceptor.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.payment.interceptor;
+
+import org.adempiere.ad.callout.spi.IProgramaticCalloutProvider;
+import org.adempiere.ad.modelvalidator.AbstractModuleInterceptor;
+import org.adempiere.ad.modelvalidator.IModelValidationEngine;
+
+public class PaymentModuleInterceptor extends AbstractModuleInterceptor
+{
+	public static final PaymentModuleInterceptor INSTANCE = new PaymentModuleInterceptor();
+
+	private PaymentModuleInterceptor()
+	{
+	}
+
+	@Override
+	protected void registerInterceptors(final IModelValidationEngine engine)
+	{
+		engine.addModelValidator(C_Payment.INSTANCE); // FRESH-348
+
+	}
+
+	@Override
+	protected void registerCallouts(final IProgramaticCalloutProvider calloutsRegistry)
+	{
+		calloutsRegistry.registerAnnotatedCallout(C_Payment.INSTANCE); // FRESH-348
```

Comment:
* C_Payment has no callout methods AFAIU
* Generally, think we should ditch the whole intercheptor; see my comment at C_Payment